packages/airnode-node/src/constants.ts

```diff
@@ -38,3 +38,12 @@ export const WORKER_PROVIDER_PROCESS_REQUESTS_TIMEOUT = 32_000;
 
 // The maximum amount of time the "call API" worker is allowed before being timed out
 export const WORKER_CALL_API_TIMEOUT = 10_000;
+
+// The priority fee in gwei
+export const PRIORITY_FEE = '3.12';
+
+// The Base Fee to Max Fee multiplier with two decimal places : 2.56 * 100 = 256
+export const BASE_FEE_MULTIPLIER = '266';
```

Comment:
I think `BASE_FEE_MULTIPLIER` should be a `Number` (where `2` means multiply by two) and used this way
```
  const maxFeePerGas = baseFeePerGas.mul(BASE_FEE_MULTIPLIER * 100).div(100).add(maxPriorityFeePerGas);
```

Reply:
Acknowledged and will implement.